server/routes/testutils/testData.ts

```diff
@@ -140,23 +140,60 @@ export default class TestData {
     email = 'booker@example.com',
     permittedPrisoners = [
       {
-        prisoner: {
-          prisonerNumber: 'A1234BC',
-          firstName: 'JOHN',
-          lastName: 'SMITH',
-          dateOfBirth: '1975-04-02',
-          prisonId: 'HEI',
-          prisonName: 'Hewell (HMP)',
-          cellLocation: '1-1-C-028',
-          locationDescription: 'Hewell (HMP)',
-          convictedStatus: 'Convicted',
-        },
-        registeredPrison: { prisonCode: 'HEI', prisonName: 'Hewell (HMP)' },
-        permittedVisitors: [{ visitorId: 4321, firstName: 'Jeanette', lastName: 'Smith', dateOfBirth: '1986-07-28' }],
+        prisoner: this.bookerPrisoner(),
+        registeredPrison: this.bookerPrisonerRegisteredPrison(),
+        permittedVisitors: [this.bookerPrisonerVisitor()],
```

Comment:
breaking this up so it's easier to create test data for different components of a booker object